### PR TITLE
chore: fix docstring and update cargo lint

### PR DESCRIPTION
**Pull request by ahramy**

### CODE REVIEW

File: packages/axelar-soroban-std/src/address.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ impl AddressExt for Address {
     /// Returns Stellar's "dead" address, represented by the constant `ZERO_ADDRESS`.
     ///
     /// # Reference
-    /// - Stellar GitHub: https://github.com/stellar/js-stellar-base/blob/master/test/unit/address_test.js
+    /// - Stellar GitHub: <https://github.com/stellar/js-stellar-base/blob/master/test/unit/address_test.js>
```
Comment (milapsheth) [inline packages/axelar-soroban-std/src/address.rs]:
```suggestion
    /// - Stellar [GitHub](https://github.com/stellar/js-stellar-base/blob/master/test/unit/address_test.js)
```
use markdown syntax

**Review by milapsheth — APPROVED** [review]

File: packages/axelar-soroban-std/src/address.rs [hunk 2]
```diff
@@ -8,7 +8,7 @@ pub trait AddressExt {
 
 impl AddressExt for Address {
     /// Returns Stellar's "dead" address, represented by the constant `ZERO_ADDRESS`.
-    ///
+    /// test
```
Comment (milapsheth) [inline packages/axelar-soroban-std/src/address.rs:11]: bad merge?

Comment (ahramy) [reply packages/axelar-soroban-std/src/address.rs]: I was testing it. removed

File: packages/axelar-soroban-std/src/address.rs (hunk 1 above)
Comment (ahramy) [reply packages/axelar-soroban-std/src/address.rs]: updated

**Pull request merged**